keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -904,6 +880,66 @@ controller, so that the endpoint slice controller doesn’t delete them.
 `EndpointSlices` will have an owner reference to their associated
 `ServiceImport`.
 
+Since a given `ServiceImport` may be backed by multiple `EndpointSlices`, a
+given `EndpointSlice` will reference its `ServiceImport` using the label
+`multicluster.kubernetes.io/service-name` similarly to how an `EndpointSlice` is
+associated with its `Service` in a single cluster.
+
+Each imported `EndpointSlice` will also have a
+`multicluster.kubernetes.io/source-cluster` label with the cluster id, a
+clusterset-scoped unique identifier for the cluster. The `EndpointSlice`s
+imported for a service are not guaranteed to exactly match the originally
+exported `EndpointSlice`s, but each slice is guaranteed to map only to a single
+source cluster.
+
+The mcs-controller is responsible for managing imported `EndpointSlice`s.
```

Comment:
This particular line doesn't sound very optional, I could be missing some context though.